fast64_internal/oot/scene/exporter/to_c/scene_table_c.py

```diff
@@ -128,28 +142,35 @@ def sceneTableToC(data, header, sceneNames, scene):
     lastSceneIdx = getInsertionIndex(sceneNames, "SCENE_TESTROOM", None, mode)
 
     # add the actual lines with the same formatting
+    # add the ifdef if this is HackerOoT
     for i in range(len(data)):
         # adds the "// Debug-only scenes"
         # if both lastScene indexes are the same values this means there's no debug scene
         if ((i - 1) == lastNonDebugSceneIdx) and (lastSceneIdx != lastNonDebugSceneIdx):
+            if isHackerOoT:
+                fileData += "\n#ifdef INCLUDE_TEST_SCENES\n"
+
             fileData += "// Debug-only scenes\n"
 
         # add a comment to show when it's new scenes
         if (i - 1) == lastSceneIdx:
-            fileData += "// Added scenes\n"
+            fileData += "\n// Added scenes\n"
 
         fileData += f"/* 0x{i:02X} */ DEFINE_SCENE("
         fileData += ", ".join(str(d) for d in data[i])
 
         fileData += ")\n"
 
+        if isHackerOoT and i == lastSceneIdx:
+            fileData += "#endif\n"
+
     # return the string containing the file data to write
     return fileData
 
 
 def getDrawConfig(sceneName: str):
     """Read draw config from scene table"""
-    fileData, header, sceneNames = getSceneTable(bpy.path.abspath(bpy.context.scene.ootDecompPath))
+    fileData, header, sceneNames, isHackerOoT = getSceneTable(bpy.path.abspath(bpy.context.scene.ootDecompPath))
```

Comment:
okay no we can't have this as a random bool going through everything, we need something better